localization/english/wc_game_rules_l_english.yml

```
@@ -4,3 +4,13 @@
  setting_wc_lore_characters_generate_families_on_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will be affected by it."
  setting_wc_lore_characters_generate_families_off:0 "Off"
  setting_wc_lore_characters_generate_families_off_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will not be affected by it."
+
+ rule_wc_interracial_marriage:0 "#Bold Warcraft#!: Interracial Marriages"
+ setting_wc_interracial_marriage_unrestricted:0 "Unrestricted"
+ setting_wc_interracial_marriage_unrestricted_desc:0 "All interracial marriages are possible, even those not normally possible."
+ setting_wc_interracial_marriage_no_modifier:0 "Modifier"
+ setting_wc_interracial_marriage_no_modifier_desc:0 "Interracial marriages will not suffer a negative modifier."
+ setting_wc_interracial_marriage_modifier:0 "Restricted"
```

Default option is highlighted.
```suggestion
 setting_wc_interracial_marriage_modifier:0 "#high Restricted#!"
```